src/components/create-appointments/DateAndTimeForm.svelte

```diff
@@ -19,11 +19,19 @@
 
 	$: {
 		if (eventType === 'once') {
+			end_date = '';
```

Comment:
check: al salvataggio nel caso in cui siamo in mode "once" la end_date deve corrispondere alla start_date